lib/config/fileconf.go

```diff
@@ -2606,23 +2596,45 @@ func (j *JamfService) toJamfSpecV1() (*types.JamfSpecV1, error) {
 		}
 	}
 	spec := &types.JamfSpecV1{
-		Enabled:      j.Enabled(),
-		Name:         j.Name,
-		SyncDelay:    types.Duration(j.SyncDelay),
-		ApiEndpoint:  j.APIEndpoint,
+		Enabled:     j.Enabled(),
+		Name:        j.Name,
+		SyncDelay:   types.Duration(j.SyncDelay),
+		ApiEndpoint: j.APIEndpoint,
+		Inventory:   inventory,
+	}
+
+	// Validate.
+	if err := types.ValidateJamfSpecV1(spec); err != nil {
+		return nil, trace.BadParameter("jamf_service %v", err)
+	}
+
+	return spec, nil
+}
+
+func (j *JamfService) toJamfCredentials() (*servicecfg.JamfCredentials, error) {
```

Comment:
`readCredentials` or `readJamfCredentials`? It's not really a "conversion" of `*JamfService`.

I don't know if we need to specify that they're jamf credentials in the method name, what else could they be?